lib/ClangImporter/ClangImporter.cpp

```diff
@@ -1519,6 +1524,7 @@ ClangImporter::emitBridgingPCH(StringRef headerPath,
   auto LangOpts = invocation.getLangOpts();
   LangOpts->NeededByPCHOrCompilationUsesPCH = true;
   LangOpts->CacheGeneratedPCH = true;
+  invocation.getHeaderSearchOpts().ValidateASTInputFilesContent = true;
```

Comment:
As above, I think we can skip this.  We can also probably skip `NeededByPCHOrCompilationUsesPCH`.